mrizwan83/bronx-warriors
# Jordy add Map feature using Leaflet

# Description
Used Leaflet library to integrate an interactive map,  based on [Leaflet Documentation](https://react-leaflet.js.org/docs/start-installation/). Ran into into some errors but used this [Youtube video](https://www.youtube.com/watch?v=jD6813wGdBA) to help display the map properly. Added basic library marker and pop up of where the football field is located.


## Main changes explained:
- User can navigate to the bottom of the page to see where the League plays
- A call to action button giving directions to where the Football field/park is located
- Library includes seamless responsiveness on smaller screens.

## How to test:
1. Check out into current branch `Jordy-Add-Map`
2. Do `npm install`
3. Do `npm run dev`  


Close #13, #8 , #14 

## Picture and Video below


https://github.com/mrizwan83/bronx-warriors/assets/95154804/9aee9b23-c31e-4ded-a1c8-30336e4a88f3




![MapFeature](https://github.com/mrizwan83/bronx-warriors/assets/95154804/1603f28c-cc98-4c50-9a5b-06c9ae93d067)
